jwst-workshop.yml

```diff
@@ -0,0 +1,258 @@
+- asdf=1.0.3=np111py35_0
+- astroid=1.4.7=py35_0
+- astroimtools=0.1.1=np111py35_0
+- astropy=1.2.1=np111py35_0
+- asv=0.1.1=np111py35_0
+- babel=2.3.4=py35_0
+- backports=1.0=py35_0
+- beautifulsoup4=4.5.1=py35_0
+- bitarray=0.8.1=py35_0
+- blaze=0.10.1=py35_0
+- bokeh=0.12.2=py35_0
+- boto=2.42.0=py35_0
+- bottleneck=1.1.0=np111py35_0
+- calcos=3.1.8=np111py35_1
+- cffi=1.7.0=py35_0
+- cfitsio=3.370=1
+- chest=0.2.3=py35_0
+- click=6.6=py35_0
+- cloudpickle=0.2.1=py35_0
+- clyent=1.2.2=py35_0
+- colorama=0.3.7=py35_0
+- configobj=5.0.6=py35_0
+- contextlib2=0.5.3=py35_0
+- costools=1.2.1=np111py35_0
+- crds=7.0.7=np111py35_0
+- cryptography=1.5=py35_0
+- cube-tools=0.0.1=np111py35_0
+- curl=7.49.0=1
+- cycler=0.10.0=py35_0
+- cython=0.24.1=py35_0
+- cytoolz=0.8.0=py35_0
+- d2to1=0.2.12=py35_0
+- dask=0.11.0=py35_0
+- datashape=0.5.2=py35_0
+- decorator=4.0.10=py35_0
+- dill=0.2.5=py35_0
+- docutils=0.12=py35_2
+- drizzlepac=2.1.6=np111py35_0
+- ds9=7.4=0
+- dynd-python=0.7.2=py35_0
+- entrypoints=0.2.2=py35_0
+- et_xmlfile=1.0.1=py35_0
+- fastcache=1.0.2=py35_1
+- fftw=3.3.4=0
+- filelock=2.0.6=py35_0
+- fitsblender=0.2.6=np111py35_0
+- flask=0.11.1=py35_0
+- flask-cors=2.1.2=py35_0
+- freetype=2.5.5=1
+- get_terminal_size=1.0.0=py35_0
+- gevent=1.1.2=py35_0
+- ginga=2.5.20161005204600=np111py35_0
+- glueviz=0.8.2=py35_1
+- greenlet=0.4.10=py35_0
+- h5py=2.6.0=np111py35_2
+- hdf5=1.8.17=1
+- heapdict=1.0.0=py35_1
+- hstcal=1.0.1=0
+- htc_utils=0.1=py35_0
+- icu=54.1=0
+- idna=2.1=py35_0
+- imagesize=0.7.1=py35_0
+- imexam=0.6.2=np111py35_0
+- ipykernel=4.5.0=py35_0
+- ipython=5.1.0=py35_0
+- ipython_genutils=0.1.0=py35_0
+- ipywidgets=5.2.2=py35_0
+- itsdangerous=0.24=py35_0
+- jbig=2.1=0
+- jdcal=1.2=py35_1
+- jedi=0.9.0=py35_1
+- jinja2=2.8=py35_1
+- jpeg=8d=2
+- jsonschema=2.5.1=py35_0
+- jupyter=1.0.0=py35_3
+- jupyter_client=4.4.0=py35_0
+- jupyter_console=5.0.0=py35_0
+- jupyter_core=4.2.0=py35_0
+- lazy-object-proxy=1.2.1=py35_0
+- libdynd=0.7.2=0
+- libgcc=4.8.5=1
+- libpng=1.6.22=0
+- libtiff=4.0.6=2
+- libxml2=2.9.2=0
+- libxslt=1.1.28=2
+- llvmlite=0.13.0=py35_0
+- locket=0.2.0=py35_1
+- lxml=3.6.4=py35_0
+- markupsafe=0.23=py35_2
+- matplotlib=1.5.3=np111py35_0
+- mistune=0.7.3=py35_1
+- mkl=11.3.3=0
+- mkl-service=1.1.2=py35_2
+- mpmath=0.19=py35_1
+- multipledispatch=0.4.8=py35_0
+- nb_anacondacloud=1.2.0=py35_0
+- nb_conda=2.0.0=py35_0
+- nb_conda_kernels=2.0.0=py35_0
+- nbconvert=4.2.0=py35_0
+- nbformat=4.1.0=py35_0
+- nbpresent=3.0.2=py35_0
+- networkx=1.11=py35_0
+- nictools=1.1.3=np111py35_0
+- nltk=3.2.1=py35_0
+- nose=1.3.7=py35_1
+- notebook=4.2.3=py35_0
+- numba=0.28.1=np111py35_0
+- numexpr=2.6.1=np111py35_0
+- numpy=1.11.1=py35_0
+- numpydoc=0.6.0=py35_0
+- odo=0.5.0=py35_1
+- openpyxl=2.3.2=py35_0
+- openssl=1.0.2j=0
+- pandas=0.18.1=np111py35_0
+- partd=0.3.6=py35_0
+- path.py=8.2.1=py35_0
+- pathlib2=2.1.0=py35_0
+- patsy=0.4.1=py35_0
+- pep8=1.7.0=py35_0
+- pexpect=4.0.1=py35_0
+- photutils=0.2.2=np111py35_1
```

Comment:
right... but I just used joe's install script that mentioned in here to do the full AstroConda install with anaconda and that's the version it pulled. I'm able to update to 0.3 though. I just changed the version in the file, pushing now